tools/validate_configs/golden_copies/expectations/versioned_blueprint/primary/main.tf

```diff
@@ -22,23 +22,23 @@ module "network" {
 }
 
 module "controller_sa" {
-  source          = "github.com/GoogleCloudPlatform/cluster-toolkit//community/modules/project/service-account?ref=v1.37.0&depth=1"
```

Comment:
Are these supposed to be updated?
The existing links don't seem to work.

Reply:
The "PR-validation" test failed on the path update to the files. This can be picked up as a separate exercise as it needs further investigation.

No changes are done to tools/validate_configs/golden_copies/ directory.